modules/canbus/proto/chassis_detail.proto

```diff
@@ -971,3 +972,200 @@ message Gem {
   optional Wiper_cmd_90 wiper_cmd_90 = 29; // report message
   optional Brake_motor_rpt_3_72 brake_motor_rpt_3_72 = 30; // report message
 }
+
+// Coolhigh vehicle starts from here
```

Comment:
Coolhigh specific definition should be moved to a separate .proto file, like coolhigh.proto.